### PR TITLE
Adds new flutter game-"wordlyPlus"

### LINKED ISSUE #344: Add flutter game - "Wordly Plus"
https://github.com/Carapacik/WordlyPlus

### PR DESCRIPTION
**Pull request by arunimaChintu**
Closes #344

### CODE REVIEW

**Comment by arunimaChintu** [comment]
@ashitaprasad Please merge this.

File: README.md [hunk 1]
```diff
@@ -154,6 +154,8 @@ All the projects added in this project are featured in [fluttergems.dev](flutter
 | Flappy Bird | [Link](https://github.com/moha-b/Flappy-Bird) | A game based on flutter animation. |
 | Quokka | [Link](https://github.com/LinwoodDev/Quokka) | Play games with/without internet. |
 | Tapit | [Link](https://github.com/webrror/tapit-flutter) | A casual finger battle game. |
+| WordlyPlus | [Link](https://github.com/Carapacik/WordlyPlus) | WordlyPlus game. |
+| Doddle | [Link](https://github.com/NaserElziadna/doddle) | Doodle drawing game. |
```
Comment (ashitaprasad) [inline README.md:158]: Remove doddle. It's already present in the list.

Comment (arunimaChintu) [reply README.md]: @ashitaprasad removed doodle,,, DOne

**Comment by arunimaChintu** [comment]
@ashitaprasad Updated...

**Pull request merged**